content/about_en.md

```diff
@@ -1,5 +1,36 @@
 **School Lunch 2050** is an educational and research project on climate change and food systems. It includes an installation and this homepage. At the moment it is focused on Japan, but we are happy to make more versions for other contexts.
 
-School Lunch 2050 was established by the **FEAST Project** and is managed by *Maximilian Spiegelberg (FEAST Project)*. The future lunch sets were created by *Yuko Kobayashi & Yuko Matsuoka (FEAST Project)*, scenario illustrations by *Elie Tanabe (田辺エリ, Sustena)*, and the web design by *Martin Heidegger*. More information can be found at the [FEAST homepage] (feastproject.org/school-lunch-2050).
+School Lunch 2050 was established by the **FEAST Project**.
 
-The School Lunch 2050 homepage has been funded by the NIHU Interactive Communication Initiative.
+_Direction:_ Maximilian Spiegelberg ([FEAST Project][feast])<br/>
+_Future lunch sets:_ Yuko Kobayashi & Yuko Matsuoka ([FEAST Project][feast]) <br/>
+_Scenario illustrations:_ Elie Tanabe (田辺エリ, [Sustena Inc.][sustena])<br/>
+_Homepage Design & Code:_ [Martin Heidegger][mh]<br/>
+
```

Comment:
_Translations:_ Yuko Kobayashi <br/>
_Research Assistance:_ Leo Porte & Haruka Tonegawa

Reply:
I added these to the changes, Is the structure as you expect?

Also: Are there links for Leo & Haruka?